internal/events/eventstream.go

```diff
@@ -426,13 +429,15 @@ func (a *eventStream) eventPoller() {
 				i1 := checkpoint[sub.info.ID]
 				i2 := sub.blockHWM()
```

Comment:
`blockHWM` returns the correct value. 
But the problem is that `i1` and `i2` are equal since we set `blockHWM` in https://github.com/hyperledger/firefly-ethconnect/blob/47a04758d131214b80843c114d139676ab215073/internal/events/eventstream.go#L407 to be the  `checkpoint` value. Hence we never commit the new HWM to disk since the logic says nothing changed.